WeightProducer/python/MCSamples_Summer20UL16.py

```diff
@@ -0,0 +1,9 @@
+import FWCore.ParameterSet.Config as cms
+
+from TreeMaker.WeightProducer.MCSample import MCSample
+
+# 13 TeV miniAOD samples - Summer20UL16
+Summer20UL16samples = [
+    # NB: amcatnlo samples have negative weight events, so NumberEvts = # positive - # negative
+    MCSample('TTToHadronic_TuneCP5_13TeV-powheg-pythia8', '106X_mcRun2_asymptotic_v13-v2', 'RunIISummer20UL16MiniAOD', 'Constant', 112592000, False),
```

Comment:
Where are all of the other samples? The same question goes for the other years.

Reply:
I still not come that point yet, I will update it.
**Actually I have a question here:**
- I thought that WeightProducer/python/MCSamples_Summer20UL16.py  file includes the all negative and positive event weight values
- So, I was stuck in the neff issue and did not finish this part yet !

Reply:
As we discussed last week, the Neff results are only needed for samples that have negative weights.